packages/language-server/src/plugins/typescript/ComponentInfoProvider.ts

```diff
@@ -57,14 +57,17 @@ export class JsOrTsComponentInfoProvider implements ComponentInfoProvider {
         return type
             .getProperties()
             .map((prop) => {
-                if (!prop.valueDeclaration) {
+                // currently there should only be one because svelte2tsx doesn't merge event dispatcher types
+                // And interface ($$Events) can't merge property type
```

Comment:
Not sure how to interpret this sentence. In #938 merging multiple decent dispatcher definitions was done. Are completions working for these, too?
Not saying we need to fix that right away if not, since this is arguably an edge case

Reply:
I tested it. Even if there're multiple declarations the type would still be correct. I think I'll remove the comment and add another test for union type.